src/SFA.DAS.FAT.Web/Models/CoursesViewModel.cs

```diff
@@ -1,22 +1,23 @@
-﻿using SFA.DAS.FAT.Domain.Courses;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Web;
+using SFA.DAS.FAT.Domain.Courses;
+using SFA.DAS.FAT.Web.Models.BreadCrumbs;
 
 namespace SFA.DAS.FAT.Web.Models
 {
-    public class CoursesViewModel
+    public class CoursesViewModel : BreadCrumbs.BreadCrumbViewModel, IShortListLink
```

Comment:
The namespace inclusion here seems redundant

Reply:
no they are to inherit the functionality